pkg/reconciler/v1alpha1/eventlistener/eventlistener.go

```diff
@@ -111,8 +74,8 @@ type Reconciler struct {
 	eventListenerLister listers.EventListenerLister
 	serviceLister       corev1lister.ServiceLister
 
-	// systemNamespace is the namespace where the reconciler is deployed
-	systemNamespace string
+	// config is the configuration passed in through flags.
```

Comment:
very minor: config is the configuration options that the Reconciler accepts. 

(We could decide to pass these in via configMaps ore something else in the future)